### PR TITLE
Fix incorrect long value restoration from strings.xml

### PR DESCRIPTION
**Pull request by ruslan-bikkinin**
Fix incorrect long value restoration from strings.xml due to android sdk bug
#264

### CODE REVIEW

File: src/android/Utilities.java [hunk 1]
```diff
@@ -59,8 +59,10 @@ public static long getApkBuildTime(Context context) {
         Long millis;
 
         try {
+            //replace double quotes needed for correct restoration of long value from strings.xml
+            //https://github.com/Microsoft/cordova-plugin-code-push/issues/264
             millis = parseLong(context.getString(
-                context.getResources().getIdentifier("CODE_PUSH_APK_BUILD_TIME", "string", context.getPackageName())
+                context.getResources().getIdentifier("CODE_PUSH_APK_BUILD_TIME", "string", context.getPackageName().replaceAll("\"",""))
```
Comment (sergey-akhalkov) [inline src/android/Utilities.java]: If I get it right, we should replace quotes for `context.getString(...)` method result, not for `context.getPackageName().replaceAll("\"","")`

Comment (sergey-akhalkov) [inline src/android/Utilities.java]: If I get it right, we should replace quotes for context.getString(...) method result, not for context.getPackageName().replaceAll("\"","")

Comment (ruslan-bikkinin) [inline src/android/Utilities.java]: yes, thanks for that

**Comment by sergey-akhalkov** [comment]
lgtm, please feel free to merge.

**Pull request merged**